libpldmresponder/bios_config.cpp

```diff
@@ -496,13 +504,14 @@ void BIOSConfig::updateBaseBIOSTableProperty()
 
 void BIOSConfig::constructAttributes()
 {
-    load(jsonDir / stringJsonFile, [this](const Json& entry) {
+    info("Bios Attribute file path: {PATH}", "PATH", (jsonDir / sysType));
```

Comment:
Does this have unwanted tracing or a single instance printing ?

Reply:
This is executed only once and its good to log this to make sure that we are fetching right bios attribute file.